stellarwp-plugin-installer.php

```diff
@@ -12,7 +12,7 @@
  */
 
 // Call our namepsace.
-namespace StellarWP\PluginInstaller;
+namespace StellarWP\StellarWPPluginInstaller;
```

Comment:
I was thinking this would have involved updating https://github.com/stellarwp/stellarwp-plugin-installer/blob/develop/composer.json#L2 instead of the namespace.

If this also includes the namespace, then make sure to update it in all files (`data.php`, `tabs.php`) as well as in the composer file itself - https://github.com/stellarwp/stellarwp-plugin-installer/blob/develop/composer.json#L14.

That being said, I don't think we should have to update the namespace, just that name in the composer file.

Reply:
That makes more sense. I missed it when I was searching, plus I've never done a plugin like this before. This is excellent knowledge collection.